core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -523,7 +500,7 @@ public String buildKeyName(long keynum) {
    * Builds a value for a randomly chosen field.
    */
   private HashMap<String, ByteIterator> buildSingleValue(String key) {
-    HashMap<String, ByteIterator> value = new HashMap<String, ByteIterator>();
+    HashMap<String, ByteIterator> value = new HashMap<>();
```

Comment:
The pom.xml has the maven-compiler-plugin set to 1.7 for source and target. I don't think we are maintaining JDK 6 compatibility anymore. We started trying to add support for JDK 9 already.